proctoru/proctoru.py

```diff
@@ -373,7 +373,8 @@ def student_view(self, context=None):
 
                     allowed_keep_old_exam = False
 
-                    if exam_start_date_time <= old_exam_time <= exam_end_date_time:
+                    if old_exam_time.tzinfo is not None  \
+                            and (exam_start_date_time <= old_exam_time <= exam_end_date_time):
```

Comment:
This line is too complicated. datetime comparison should always be simple to read for human.

I simply do not understand this statement whithout reading Python doc:

    exam_start_date_time <= old_exam_time <= exam_end_date_time

I propose to split it in two, please verify this is equivalent:

    if old_exam_time.tzinfo and \
        exam_start_date_time <= old_exam_time and \
        exam_start_date_time <= exam_end_date_time: